index.js

```diff
@@ -91,19 +114,23 @@ export function getSupportedBiometryType(): Promise {
  * @param {string} server URL to server.
  * @param {string} username Associated username or e-mail to be saved.
  * @param {string} password Associated password to be saved.
+ * @param {string} minimumSecurityLevel `SECURITY_LEVEL` defines which security
+ *                 level is minimally acceptable for this password.
  * @param {object} options Keychain options, iOS only
  * @return {Promise} Resolves to `true` when successful
  */
 export function setInternetCredentials(
   server: string,
   username: string,
   password: string,
+  minimumSecurityLevel?: SecMinimumLevel,
```

Comment:
this is a breaking change (https://github.com/oblador/react-native-keychain/pull/176/files#r243020977); can you move this to `Options`?

Reply:
done